Fixed typos and improved style of css tutorial

captainbrosset opened the pull request:
There were a couple of typos on this page that I fixed.
I used this opportunity to improve the score reported by acrolinx (from 89 to 95), by changing some of the style.

Fixes #1343.

opbld33 commented:
Docs Build status updates of commit _[8a7d3e7](https://github.com/captainbrosset/edge-developer/commits/8a7d3e73da058b9dc983bcb1755f3e6d71e244f8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/devtools-guide-chromium/beginners/css.md](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/beginners/css.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C19%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202107190850301501-1347%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-19T02%3A15%3A57Z&ske=2021-07-26T02%3A20%3A57Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-19T08%3A46%3A12Z&se=2021-08-19T08%3A51%3A12Z&sr=b&sp=r&sig=phI%2ByojMtaNdtU4b4erfTitsNSF%2BNAycul4J1GI2eTA%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

mikehoffms commented on microsoft-edge/devtools-guide-chromium/beginners/css.md:
```diff
@@ -26,7 +26,7 @@ keywords: microsoft edge, web developement, f12 tools, devtools
 
```
In the screenshot that's below "Choose New Style Rule (New Style Rule) which is next to .cls. An empty ruleset appears with a as the selector.", add red box around +.   eg: 
1. in VS Code, with doc tool extension installed, click the image link to display image in VS Code.  
2. shortcut Ctrl+Alt+R opens File Explorer w/ the file selected.
3. Right-click, Edit (in Paint).  
4. Zoom (Ctrl+PgUp).  3px wide red box. 
5. Save.
I jumped into that section w/o context, and it took a long time to figure out what the sentence was talking about in the screenshot.  Good example of why red boxes are required, for dense UI.

mikehoffms commented on microsoft-edge/devtools-guide-chromium/beginners/css.md:
```diff
@@ -590,11 +590,11 @@ Complete the following actions to bootstrap and provide classes for aligning ele
 ## Next steps  
 
-Congratulations, you are done.  
+Congratulations, you're done.  
```
```suggestion
Congratulations, you're done!
```
Suggested for natural tone in this context.  https://docs.microsoft.com/en-us/style-guide/punctuation/exclamation-points

mikehoffms reviewed (changes requested):
Edit pass.  Suggested changes, largely to make the tone more natural.

opbld32 commented:
Docs Build status updates of commit _[ceddaca](https://github.com/captainbrosset/edge-developer/commits/ceddaca61d912db63224b271af09d636e7847232)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/devtools-guide-chromium/beginners/css.md](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/beginners/css.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |
[microsoft-edge/devtools-guide-chromium/media/beginners-css-rule1.msft.png](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/media/beginners-css-rule1.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C26%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202107260953107376-1347%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-23T09%3A01%3A59Z&ske=2021-07-30T09%3A06%3A59Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-26T09%3A48%3A52Z&se=2021-08-26T09%3A53%3A52Z&sr=b&sp=r&sig=rTf3XNuIn5wAo%2FELYZl6%2BVIUAZF3xoySM140BaePv4Q%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

captainbrosset commented:
Thank you @mikehoffms, I believe I addressed all your comments. I agree the tone of this tutorial was very robotic.
I also changed one of the class names used in the code from `custom-background` to `my-class-name` to make it more obvious that this is up to users to choose, and to avoid making the class name too specifically tied to what it actually changes.

opbld31 commented:
Docs Build status updates of commit _[b1560d0](https://github.com/captainbrosset/edge-developer/commits/b1560d09facadb3cc0684c2394769a3072502a69)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/devtools-guide-chromium/beginners/css.md](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/beginners/css.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |
[microsoft-edge/devtools-guide-chromium/media/beginners-css-add1.msft.png](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/media/beginners-css-add1.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |
[microsoft-edge/devtools-guide-chromium/media/beginners-css-add4.msft.png](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/media/beginners-css-add4.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |
[microsoft-edge/devtools-guide-chromium/media/beginners-css-edit1.msft.png](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/media/beginners-css-edit1.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |
[microsoft-edge/devtools-guide-chromium/media/beginners-css-rule1.msft.png](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/media/beginners-css-rule1.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C26%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202107261004306029-1347%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-26T03%3A06%3A48Z&ske=2021-08-02T03%3A11%3A48Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-26T10%3A00%3A12Z&se=2021-08-26T10%3A05%3A12Z&sr=b&sp=r&sig=YhOyOsVa7DIvbYf26atflW3198Qh%2BBqMPRKt0eryDPo%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld33 commented:
Docs Build status updates of commit _[0a53ecc](https://github.com/captainbrosset/edge-developer/commits/0a53ecc9839767bfb170e9f8836093aa9b24541d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/devtools-guide-chromium/beginners/css.md](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/beginners/css.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |
[microsoft-edge/devtools-guide-chromium/media/beginners-css-add1.msft.png](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/media/beginners-css-add1.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |
[microsoft-edge/devtools-guide-chromium/media/beginners-css-add4.msft.png](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/media/beginners-css-add4.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |
[microsoft-edge/devtools-guide-chromium/media/beginners-css-edit1.msft.png](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/media/beginners-css-edit1.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |
[microsoft-edge/devtools-guide-chromium/media/beginners-css-rule1.msft.png](https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/ruleset-typo/microsoft-edge/devtools-guide-chromium/media/beginners-css-rule1.msft.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css?branch=pr-en-us-1347) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C7%5C27%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202107272338422718-1347%5Cworkflow_report.html?skoid=5f458e1a-fa67-4ae6-a972-54ea42ec670e&sktid=975f013f-7f24-47e8-a7d3-abc4752bf346&skt=2021-07-26T02%3A11%3A44Z&ske=2021-08-02T02%3A16%3A44Z&sks=b&skv=2020-04-08&sv=2020-04-08&st=2021-07-27T23%3A34%3A35Z&se=2021-08-27T23%3A39%3A35Z&sr=b&sp=r&sig=drRIUHsszVFSibMOKYyHcfYAPjko6aYgnDxiLFvjpB4%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
